tests/Doctrine/Tests/Common/DataFixtures/Purger/ORMPurgerTest.php

```diff
@@ -54,4 +55,18 @@ public function testTableNameWithSchema()
         $tableName = $method->invokeArgs($purger, [$metadata, $platform]);
         $this->assertStringStartsWith('test_schema', $tableName);
     }
+
+    public function testGetTableNameQuoted()
```

Comment:
```suggestion
    public function testGetTableNameQuoted() : void
```

Reply:
I agree in principle with this, but the other methods on the class don't have return types - prefer consistency in this PR and then update all test class methods in another to include return types?

Reply:
> update all test class methods in another to include return types

That will mean a bigger PR than it should. Consistency is great, but consistency with deprecated things isn't.

Reply:
Good argument - changed